test/bn_vectors.js

```diff
@@ -76,6 +76,9 @@ sjcl.test.vector.bn_powermod = [
   // // This was disabled because it's slow (~2.5s with `rhino -O 9`). Once
   // // a better powermod algorithm is implemented (e.g., using Montgomery
   // // reduction), this can be re-enabled.
+  //
+  // // Note:  This example can't be done with Montgomery reduction anyways
```

Comment:
@Nilos A lot of the tests are already verifying `montpowermod` works because of [this](https://github.com/Bren2010/sjcl/blob/bn-fix2/core/bn.js#L292-297) (and a couple of them are not going that path and testing the regular `powermod` code, as well).